### ISSUE TITLE
Getting in touch - automated builds

### ISSUE THREAD

**Issue by jeffersonn-1** [Pro/dkim-exchange#234]
Hey there...
I may be able to put together an automated build process as new CU comes out. That's probably about the most that I could contribute. 
I was thinking about it a few years ago and here we are today.
Kindly get in touch -- I'm reachable on LinkedIn.
Jefferson

**Comment by Pro** [comment]
Hi jefferson,
thanks for your suggestion. As mentioned in the readme I currently do not have the time to fully maintain this repo, so I'm happy for any help and automated build processes, especially automatic deployment.

I already implemented travis, so maybe we can use that to create automatic deployment for any new tags and commits for all the exchange versions.

I did not find you on linkedin, but feel free to contact me directly here: http://profanter.me/contact
